hapic/ext/pyramid/context.py

```diff
@@ -181,3 +182,12 @@ def serve_directory(
             name=route_prefix,
             path=directory_path,
         )
+
+    def _add_exception_class_to_catch(
+        self,
+        exception_class: typing.Type[Exception],
+        http_code: int,
+    ) -> None:
+        self._handled_exception_and_http_codes.append(
```

Comment:
I would prefer to use a specific class instead of a tuple:

```
class HandledException(object):
    def __init__(self, exception_class, http_code):
        self.exception_class = exception_class
        self.http_code = http_code
```

Reply:
By the way, the attribute `_handled_exception_and_http_codes` would be renamed to `_handled_exceptions`

Reply:
done